airflow/bin/cli.py

```diff
@@ -146,19 +152,19 @@ def trigger_dag(args):
 def variables(args):
     if args.get:
         try:
-            var = Variable.get(args.get,
-                               deserialize_json=args.json,
-                               default_var=args.default)
+            var = client.get_var(args.get,
+                                 deserialize_json=args.json,
+                                 default_var=args.default)
             print(var)
         except ValueError as e:
             print(e)
     if args.set:
-        Variable.set(args.set[0], args.set[1])
+        # and serialize_json as an option?
```

Comment:
I lefted out serialize_json as a option because CLI input is already str type and will be serialized as a string by the JSON encoder, which is then deserialized into str/unicode instead of a dict.